lib/topological_inventory/ansible_tower/operations/worker.rb

```diff
@@ -35,7 +38,9 @@ def run
         def process_message(message)
           Processor.process!(message)
         rescue StandardError => err
-          logger.error("#{err}\n#{err.backtrace.join("\n")}")
+          model, method = message.message.to_s.split(".")
+          task_id = message.payload&.fetch_path('params','task_id')
+          logger.error("#{model}##{method}: Task(id: #{task_id}) #{err.cause}\n#{err}\n#{err.backtrace.join("\n")}")
```

Comment:
@lpichler can we check if this will  be handled by the insights-api-common here
https://github.com/RedHatInsights/insights-api-common-rails/blob/f0b86e82f870bb36b798a1dbccbb73edf52f8b5f/lib/insights/api/common/application_controller_mixins/exception_handling.rb#L10
If it does then we will have 2 log lines doing the same thing
Otherwise looks good

Reply:
@mkanoor this is not Rails app, so handler ending with `render :json` can't be used. All collector/operations repos are not using insights-api-common, but providers-common gem, where it's worth to move these common methods